### PR TITLE
add recent components for compiler comparison

### PR DESCRIPTION
**Pull request by salmma**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -67,15 +74,31 @@ services:
...
+
+      # activate correct application properties
+      SPRING_PROFILES_ACTIVE: docker
+
+      # Configure QPROV connection
+      QPROV_HOSTNAME: qprov
+      QPROV_PORT: 8080
+      TRANSLATOR_HOSTNAME: quantum-transpiler
+      TRANSLATOR_PORT: 5000
+
```
Comment (lharzenetter) [inline docker-compose.yml:93]: maybe remove the second empty line here

File: docker-compose.yml [hunk 2]
```diff
@@ -108,17 +151,103 @@ services:
       - exec_data:/data
     depends_on:
       - redis
-    deploy:
```
Comment (lharzenetter) [inline docker-compose.yml:111]: was this on purpose?

File: README.md [hunk 3]
```diff
@@ -4,32 +4,64 @@
 
 Docker Compose file for running the entire QuAntiL environment.
 
-The fastest way to get started is using [Docker Compose](https://docs.docker.com/compose/):
+The fastest way to get started is using [Docker Compose](https://docs.docker.com/compose/).  
+[Profiles](https://docs.docker.com/compose/profiles/) are used to run different feature sets.  
 
+Defined feature sets are:
```
Comment (lharzenetter) [inline README.md]: Please add some more details here: what is a feature set? what do i need profiles for? can i start it without any feature set/profile? what happens then?

**Review by lharzenetter — CHANGES_REQUESTED** [review]
LGTM besides some minor documentation enhancements

File: docker-compose.yml (hunk 2 above)
Comment (salmma) [reply docker-compose.yml:111]: Yes, only a single worker should run as 4 is a bit of an overhead.

Comment (salmma) [reply docker-compose.yml:111]: ok, I would leave it in but only with a single replica to keep in mind that it is possible defining multiple workers :D

File: README.md (hunk 3 above)
Comment (salmma) [reply README.md]: I hope it is better now

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**